### ISSUE TITLE
Disable delete button on stacked inline

### ISSUE THREAD

**Issue by MathiasKowoll** [otto-torino/django-baton#147]
Hi,
I have an inline model and I have can_delete=False, but delete link still appears. Is there any way to disable delete button without modifying admin template?

**Comment by MathiasKowoll** [comment]
I also noted that when adding a new row there is a button, when editing existing inline there is checkbox.

**Comment by abidibo** [comment]
Hi @MathiasKowoll ,
I don't know if I have understood correctly, but I can't reproduce any bug here.

Adding `can_delete=False` to an admin inline makes the delete checkbox disappear, and actually, you can't delete existing items anymore. But still, you can add and remove newly added items:

![Screenshot from 2021-04-07 09-31-30](https://user-images.githubusercontent.com/444093/113828171-615e9d80-9784-11eb-9847-d4c043599f69.png)

As for the second comment, that's standard for default Django admin: checkbox for existing inlines, icon for new inlines:

![Screenshot from 2021-04-07 09-35-17](https://user-images.githubusercontent.com/444093/113828426-a7b3fc80-9784-11eb-8c93-3eccfdb9ee88.png)

**Comment by MathiasKowoll** [comment]
It would be great if we can have a consistent view with the delete button. As my first Question I figure it out how it works and works ok.

Thanks

**Comment by abidibo** [comment]
Hi @MathiasKowoll ,

I won't do that because it goes against normal Django behaviour. Also, it will not be clear as UX. The fact is that using a button instead of a checkbox, the user would see an inline element disappear, thinking it was actually deleted, but this is not true! In fact actually the inline will be deleted only after the form submission and that is confusing. At the same time, it would be impossible to use a checkbox for new created inlines, it's non-sense, in that case since you haven't already saved the inlines, make them disappear immediately is the right thing to do, IMO.

**Comment by MathiasKowoll** [comment]
Thx for you reply
